duqtools/config.py

```diff
@@ -162,3 +162,6 @@ def __new__(cls, *args, **kwargs):
         if not Config._instance:
             Config._instance = object.__new__(cls)
         return Config._instance
+
+
+cfg = Config()
```

Comment:
if the `from duqtools.config import cfg` does copy the cfg object, then we could also turn cfg into a property:

```suggestion
@property
def cfg():
    return Config()
```